script/zonemaster_backend_testagent

```diff
@@ -149,30 +149,32 @@ sub main {
 
         my $fetch_test_timer = [ gettimeofday ];
 
-        my $id = $self->db->get_test_request( $self->config->ZONEMASTER_lock_on_queue );
+        my ( $test_id, $batch_id ) = $self->db->get_test_request( $self->config->ZONEMASTER_lock_on_queue );
         $self->db->process_unfinished_tests(
             $self->config->ZONEMASTER_lock_on_queue,
             $self->config->ZONEMASTER_max_zonemaster_execution_time,
         );
 
         Zonemaster::Backend::Metrics::timing("zonemaster.testagent.fetchtests_duration_seconds", tv_interval($fetch_test_timer) * 1000);
 
-        if ( $id ) {
-            $log->info( "Test found: $id" );
-            if ( $self->pm->start( $id ) == 0 ) {    # Forks off child process
-                $log->info( "Test starting: $id" );
+        my $show_progress = defined $batch_id ? 0 : 1;
+
+        if ( $test_id ) {
+            $log->info( "Test found: $test_id" );
```

Comment:
Maybe replace `info()` with `infof()` while we're at it?

Reply:
I replaced the next occurrences as well.